network-api/networkapi/settings.py

```diff
@@ -499,6 +499,23 @@
         }
     }
 
+
+# Wagtail Embeds
+
+datawrapper_oembed = {
```

Comment:
since this is a constant, let's call it `DATAWRAPPER_PROVIDER` with the corresponding update on L515

Reply:
Good call 👍